src/status_im/contexts/communities/events.cljs

```diff
@@ -177,3 +178,13 @@
                      :params     []
                      :on-success #(rf/dispatch [:communities/fetched-collapsed-categories-success %])
                      :on-error   #(log/error "failed to fetch collapsed community categories" %)}]}))
+
+
+(rf/reg-event-fx :communities/share-community-channel-url-with-data 
+    (fn [_ [chat-id]]
+      (let [community-id (subs chat-id 0 constants/community-id-length)
+            channel-id (subs chat-id constants/community-id-length)] 
+   {:json-rpc/call [{:method     "wakuext_shareCommunityChannelURLWithData"
+                     :params      [{:CommunityID community-id :ChannelID channel-id}] 
+                     :on-success  #(share/open {:url %}) 
```

Comment:
As per the designs, the share menu needs to have a title as `Channel on Status` and needs to have a sub-title which is the url (FYI, this is supported only on iOS). You can check the format in `status-im.contexts.wallet.account.tabs.about.view` (this one doesn't have a sub-title tho)

Reply:
I am still waiting on access to the designs. I was unable to find a `subtitle` option in the [docs](https://react-native-share.github.io/react-native-share/docs/share-open#activityitemsources-ios-only). 

I pushed the changes to use `ActivityItemSources`. Let me know if they are not matching what the design expects.